src/main/java/com/es/phoneshop/model/cart/Cart.java

```diff
@@ -1,14 +1,15 @@
 package com.es.phoneshop.model.cart;
```

Comment:
I don't see a link to the cart page on UI. This is a bit of a non-intuitive UX.

Reply:
I don't see a way to go to the cart page directly from the product list page. How can the user get to the cart page besides entering http://localhost:8080/phoneshop-servlet-api/cart in the browser address bar directly?

Reply:
Ohhhh, i get it .  i'll fix it in next chapter